docs/man/borg-compact.1

```diff
@@ -48,7 +48,7 @@ Depending on the amount of segments that need compaction, it may take a while,
 so consider using the \fB\-\-progress\fP option.
 .sp
 A segment is compacted if the amount of saved space is above the percentage value
-given by the \fB\-\-threshold\fP option. If ommitted, a threshold of 10% is used.
+given by the \fB\-\-threshold\fP option. If omitted, a threshold of 10% is used.
```

Comment:
the man pages are auto-generated. a lot of this comes from `src/borg/archiver.py` - can you check/change there?

Reply:
> the man pages are auto-generated. a lot of this comes from `src/borg/archiver.py` - can you check/change there?

Yeap, also fixed there!

Reply:
ok

Reply:
i reverted this change in the autogenerated file.